samples/25.message-reaction/src/main/java/com/microsoft/bot/sample/messagereaction/ActivityLog.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.sample.messagereaction;
+
+import com.microsoft.bot.builder.Storage;
+import com.microsoft.bot.schema.Activity;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+public class ActivityLog {
+
+    private Storage _storage;
```

Comment:
not sure about variables beginning with underscore. Maybe the other guys can say more about this

Reply:
change it to remove the underscore

Reply:
Yes, Java doesn't manage underscore for properties